frontend/ui/data/keyboardlayouts/en_keyboard.lua

```diff
@@ -55,73 +57,151 @@ local _Y_ = en_popup._Y_
+            { _P_, _p_, pl2, pls, },
         },
-        -- third row
-        {  --  1       2       3       4       5       6       7       8
-            { "",   "",   "2/2",  "1/2",   "",   "",   "",    "",
-              width = 1.5
-            },
-            { _Z_,    _z_,    "&",    "7",    "Á",    "á",    "Ű",    "ű", },
-            { _X_,    _x_,    "*",    "8",    "Ø",    "ø",    "Ã",    "ã", },
-            { _C_,    _c_,    "£",    "9",    "Í",    "í",    "Þ",    "þ", },
-            { _V_,    _v_,    "<",    com,    "Ñ",    "ñ",    "Ý",    "ý", },
-            { _B_,    _b_,    ">",    prd,    "Ó",    "ó",    "†",    "‡", },
-            { _N_,    _n_,    "‘",    "↑",    "Ú",    "ú",    "–",    "—", },
-            { _M_,    _m_,    "’",    "↓",    "Ç",    "ç",    "…",    "¨", },
+        -- Third row
+        { --   R    r    S    s
+            { _A_, _a_, ls2, ls1, },
+            { _S_, _s_, mr2, mr1, },
+            { _D_, _d_, dgr, "(", },
+            { _F_, _f_, tpg, ")", },
+            { _G_, _g_, hs2, hsh, },
+            { _H_, _h_, pd2, pdc, },
+            { _J_, _j_, _1s, _1n, },
+            { _K_, _k_, _2s, _2n, },
+            { _L_, _l_, _3s, _3n, },
+            { sec, cse, sec, cse, }, -- comma/semicolon with CSS popup block
+        },
+        -- Fourth row
+        { --   R    r    S    s
+            { label = "",
+              width = 1.5, },           -- Shift
+            { _Z_, _z_, prm, "{", },
+            { _X_, _x_, "°", "}", },
+            { _C_, _c_, "«", "[", },
+            { _V_, _v_, "»", "]", },
+            { _B_, _b_, _0s, _0n, },
+            { _N_, _n_, "↑", "↑", },
+            { _M_, _m_, "↓", "↓", },
             { label = "",
-              width = 1.5,
-              bold = false
-            },
+              width = 1.5, },           -- Backspace
         },
-        -- fourth row
-        {
-            { "Sym",  "Sym",  "ABC",  "ABC",  "Sym",  "Sym",  "ABC",  "ABC",
-              width = 1.5},
-            { label = "🌐", },
-            { "Äéß",  "Äéß",  "Äéß",  "Äéß",  "Äéß",  "Äéß",  "Äéß",  "Äéß", },
-            { label = "space",
-              " ",    " ",    " ",    " ",    " ",    " ",    " ",    " ",
-              width = 3.0},
-            { com,    com,    "“",    "←",    "Ũ",   "ũ",    com,    com, },
-            { prd,    prd,    "”",    "→",    "Ĩ",   "ĩ",    prd,    prd, },
-            { label = "⮠",
-              "\n",   "\n",   "\n",   "\n",   "\n",   "\n",   "\n",   "\n",
+        -- Fifth row
+        { --   R    r    S    s
+            { label = "⌥",
               width = 1.5,
-              bold = true
-            },
+              bold = true,
+              alt_label = "SYM"},
+            { label = "🌐", },
+            { cop, pco, cop, pco, },    -- period/colon with RegEx popup block
+            { label = "_",              -- Spacebar
+              " ", " ", " ", " ",
+              width = 3.0, },
```

Comment:
So, we're going with a symbol (and an underscore)  on the space key ?
Isn't it obvious, with a width=3, that's it is the space key ? :)
(I had it removed in the fr_keyboard, so I'll be fine :)

Reply:
OK, I removed the sign. Some other layouts though replaced it with the actual name in their language

Reply:
May be that was just by mimicry because the en_keyboard has it named "space" ?

Reply:
Yes, I think this was back from days when keyboard layouts were made based on the EN layout that had "space" label. I don't know, should we remove the label in other layouts too, or leave it for those who change them?

Reply:
Dunno. I'd say let the other ones you haven't yet touched untouched - so we know easily those who need the update.
I assume all the ones you modified are just fine ? Or do they still need some small/bigger rework from native users? You might want to summarize all that when we're done with this PR :)

Reply:
> I assume all the ones you modified are just fine

Sorry, I somehow don't get proper focus to such commented discussion when I tap notifications. 
Yes, all those that are present in this PR are just fine. They are not broken/skewed and work as expected. 

> Or do they still need some small/bigger rework from native users?

This is only true for those I didn't touch at all. Mostly because they do not copy EN layout and are written from scratch manually. So they are not affected by the new design changes at all (except visuals and button behavior).

> You might want to summarize all that when we're done with this PR :)

I'll clearly state what layouts need complete re-work in the OP.